.gitignore

```diff
@@ -0,0 +1,3 @@
+composer.lock
```

Comment:
why do you ignore `composer.lock`?

Reply:
Because it's a library, so when used as a dependency, [the lock file is ignored](https://getcomposer.org/doc/02-libraries.md#lock-file). To me, it makes sense to test it alone the same way. Ideally, there should be one more build with `composer update --prefer-lowest` to make sure the entire range of supported versions is actually supported.